test/crio-wipe.bats

```diff
@@ -32,35 +32,27 @@ function run_crio_wipe() {
 # test_crio_wiped_containers checks if a running crio instance
 # has no containers or pods
 function test_crio_wiped_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
-
-	run crictl ps -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
+	output=$(crictl pods -v)
+	[ "$output" = "" ]
```

Comment:
is this checking equality?

Reply:
😳

Reply:
Yes, in bash, `[` is equal to `test` built-in, and `=` in test is checking equality. Quoting `help test`:

```
      STRING1 = STRING2
                     True if the strings are equal.
```

Now, `==` is probably bash extention.